.gitmodules

```diff
@@ -1,5 +1,5 @@
 [submodule "sw_src"]
 	path = sw_src
 	url = https://github.com/DrylandEcology/SOILWAT2.git
-	branch = master
+	branch = enhance_petfunc
         ignore = dirty
```

Comment:
Don't forget to update to master after this branch of SOILWAT2 is folded in.

Reply:
done with commit e6f9110c53ab1510f091b5aaf9ef132e081225ad